pkgs/tools/audio/tts/default.nix

```
@@ -114,6 +115,18 @@ python3Packages.buildPythonApplication rec {
     "test_parametrized_gan_dataset"
   ];
 
+  preCheck = ''
+    # use the installed TTS in $PYTHONPATH instead of the one from source to also have cython modules.
+    rm -rf TTS
```

```suggestion
    rm -r TTS
```

We want to notice when it is gone.